test/tests/table.ts

```diff
@@ -117,14 +117,20 @@ suite('Table', () => {
     })
 
     suite('query', () => {
+        test('should allow you to chain the query statements', async () => {
+            const tableCursor = decentiumTrendingTable.query({from: 5, to: 10}).query({to: 8})
```

Comment:
What's the use case for this? 🤔

Reply:
It was requested by Shaq during one of the Web Client SDK Weekly calls. It basically allows you to narrow down the query the same way you would be able to narrow down queries using a typical ORM.

Reply:
I think the typical use case is probably going to be:

```
const relevantEntries = table.query({ scope })

const relevantEntriesInSpecificRange = relevantEntries.query({ from: 'something' })
const relevantEntriesInDifferentRange = relevantEntries.query({ from: 'something-else' })
```

Reply:
I should probably add a unit test checking that that exact use case works though 😅

Reply:
They won't work - since changing `from` is just going to alter the `lower_bounds` on the query. 

We'll need to build something more advanced, something that collects rows from the API and then filters internally, if we want to achieve something like this. That being said, I think if we want this functionality, it goes somewhere else (like a query builder) and not on the cursor.

Reply:
I think we may be thinking of two different things 😅  What I meant by combining `query()` statements is basically a way to update some of the fields without needing to redefine all of them (like in my example above where I specified the `from` field in two different places without needing to specify the `scope` field every time. Are you talking about having a query builder that allows you to combine two separate fields to query by a multi field index?

Reply:
Yes, that query builder concept I think is what Shaq was describing on that call - so that's what I was talking about.

For what you're showing here, I don't see the value in being able to redefine those things and I can't imagine a scenario where I'd actually do that 😅 

I suppose it just works that way though - so its there if someone needs it!

Reply:
I do remember us specifically discussing being able to define a `scope` in one query and a `from` or `to` statement in another one 😄 I do agree that a query builder would probably be more valuable although that would be more of an undertaking.